### DISCUSSION TITLE
Gridpoints API current conditions

### DISCUSSION THREAD

**Discussion by mrgatesjunior** [weather-gov/api#227] [Q&A]
I like the gridpoints API in general and like also the hourly function.  However, unless I am missing something, neither provide current conditions.  Meaning current temperature, wind speed, etc.

How does one even pull current conditions with the gridpoints API approach ?

**Comment by netbymatt** [comment]
It's a few steps. First get a list of stations such as
https://api.weather.gov/gridpoints/LOT/52,80/stations
The list seems to be sorted with the closest station first.
Once you grab the station URL, get the latest observations such as:
https://api.weather.gov/stations/KDPA/observations/latest

**Comment by mrgatesjunior** [comment]
> It's a few steps. First get a list of stations such as
> https://api.weather.gov/gridpoints/LOT/52,80/stations
> The list seems to be sorted with the closest station first.
> Once you grab the station URL, get the latest observations such as:
> https://api.weather.gov/stations/KDPA/observations/latest

Yeah, I figured that out.  However, it would be nice to be able to pull everything from one call through the gridpoints API.

**Comment by scadergit** [comment]
The API is currently designed out of necessity for efficiency and performance. Future environment improvements might allow graphql or other convienences.

**Comment by katilley** [comment]
Would it be possible to create a new feature request for grid based current Conditions? Beyond the multiple steps of getting station observations, some stations don't have data (not sure if they are down or whatever). For example: https://api.weather.gov/stations/SFOC1/observations

So you'd need to keep trying stations until you found one with all the data. 

It would also be nice for the current conditions (either through the new API or existing observations) to have apparentTemperature (rather than just heatIndex and windChill). Is apparentTemperature just either heatIndex or windChill, whichever is present?
